bundles/org.openhab.binding.wundergroundupdatereceiver/README.md

```diff
@@ -0,0 +1,115 @@
+# Wunderground Update Receiver Binding
+
+Many personal weather stations are only capable of submitting measurements to the wunderground.com update site.
+
+This binding enables acting as a receiver of updates from weather stations that post measurements
+to https://rtupdate.wunderground.com/weatherstation/updateweatherstation.php. If the hostname is configurable - as on
+weather stations based on the Fine Offset Electronics WH2600-IP - this is simple, otherwise you have to set up dns
+such that it resolves the above hostname to your server, without preventing the server from resolving the proper ip if
+you want to forward the request.
```

Comment:
```suggestion
This binding enables acting as a receiver of updates from weather stations that post measurements
to https://rtupdate.wunderground.com/weatherstation/updateweatherstation.php. 
If the hostname is configurable - as on weather stations based on the Fine Offset Electronics WH2600-IP - this is simple, otherwise you have to set up dns such that it resolves the above hostname to your server, without preventing the server from resolving the proper ip if you want to forward the request.
```